### ISSUE TITLE
Dependencies does not support PHP 8.2

### ISSUE THREAD

**Issue by malandles** [saloonphp/xml-wrangler#30]
Using `composer require saloonphp/xml-wrangler`, it tries to install dependencies that are locked to PHP 8.1.

See results below:
```
./composer.json has been updated
Running composer update saloonphp/xml-wrangler
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - saloonphp/xml-wrangler[v1.0.0, ..., v1.2.1] require veewee/xml ^2.11.2 -> satisfiable by veewee/xml[2.11.2, 2.12.0, 2.13.0, 2.14.0].
    - saloonphp/xml-wrangler v0.2.3 requires veewee/xml ^2.11.1 -> satisfiable by veewee/xml[2.11.1, ..., 2.14.0].
    - saloonphp/xml-wrangler[v0.2.0, ..., v0.2.2] require veewee/xml ^2.11 -> satisfiable by veewee/xml[2.11.0, ..., 2.14.0].
    - saloonphp/xml-wrangler[v0.0.2, ..., v0.1.1] require veewee/xml ^2.9 -> satisfiable by veewee/xml[2.9.0, ..., 2.14.0].
    - saloonphp/xml-wrangler v0.0.1 requires veewee/xml ^2.7 -> satisfiable by veewee/xml[2.7.0, ..., 2.14.0].
    - veewee/xml[2.7.0, ..., 2.14.0] require azjezz/psl ^2.0.3 -> satisfiable by azjezz/psl[2.0.3, ..., 2.9.1].
    - azjezz/psl 2.0.3 requires php ~8.1.0 -> your php version (8.2.21) does not satisfy that requirement.
    - Root composer.json requires saloonphp/xml-wrangler * -> satisfiable by saloonphp/xml-wrangler[v0.0.1, ..., v0.2.3, v1.0.0, ..., v1.2.1].
```

**Comment by fredbradley** [comment]
I believe this would be fixed if my PR was accepted and merged. #25 (Would fix 8.2 and 8.3)

**Comment by Sammyjo20** [comment]
Fixed in #25
